sql/moz-fx-data-marketing-prod/google_search_console_derived/site_impression_exports_empty_check_v1/query.sql

```diff
@@ -0,0 +1,46 @@
+{% set gsc_export_dataset_ids = [
```

Comment:
Is there a reason why we're doing this via query.sql rather than ETL checks using `checks.sql`?

Reply:
I addressed that question in an [earlier comment](https://github.com/mozilla/bigquery-etl/pull/4892#discussion_r1468022013).  There are no upstream ETLs to have `checks.sql` files for.  And even if we created intermediate upstream ETLs to put checks on, that wouldn't work to actively wait for the source data to become available like this does, because when checks fail things just stop and don't retry.